cordova_in_app_browser.mli

```diff
@@ -204,3 +203,39 @@ val open_ :
   unit
 [@@js.global "cordova.InAppBrowser.open"]
 (* -------------------------------------------------------------------------- *)
+
+(*-------------------------------TODO-------------------------------------------- *)
+type t
+
+val open_t :
+  url:string ->
+  tgt:target ->
+  option:string ->
+  t
+[@@js.builder]
+
+val url : t -> string
+val tgt : t -> target
+val option : t -> string
+
+(*[@@@js.implem*)
+
+open Js_of_ocaml.Dom_html
+val addEventListener :
+  t                                -> (*Where to add the listener*)
+  string        -> (*The event type*)
+  (event -> unit)                  -> (*The call_back*)
```

Comment:
@dannywillems Hello Danny !
Would you know whether/how one can use a type like `Js_of_ocaml.Dom_html.event` in gen_js_api ?

Reply:
I don't know if I understand correctly the question, but if the function is expecting an event, simply using `Js_of_ocaml.Dom_html.event` should be enough shouldn't?

Reply:
Unfortunately things are not that simple. Simply using `Js_of_ocaml.Dom_html.event` will result in `gen_js_api` generating `Js_of_ocaml.Dom_html.event_of_js`.

So it seems that one would need to supply a transformation function. I've tried all kinds of stuff already, but to no avail yet.

Reply:
Using Ojs.t?

Reply:
Haven't tried, ugly, and not sure it is the best idea:
```
type event = Ojs.t

[@@@js.stop]

val of_dom_html : Js_of_ocaml.Dom_html.event -> event

val to_dom_html : event -> Js_of_ocaml.Dom_html.event

[@@@js.start]

[@@@js.implem
let of_dom_html x = Obj.magic x

let to_dom_html x = Obj.magic x]

val addEventListener : t -> string -> (event -> unit) -> unit [@@js.call]
```